deploy/olm-catalog/service-telemetry-operator/manifests/service-telemetry-operator.clusterserviceversion.yaml

```diff
@@ -90,6 +90,7 @@ metadata:
       the required dependencies and configuration of various components to build a
       Service Telemetry platform for telco grade monitoring.
     olm.skipRange: '>=<<BUNDLE_OLM_SKIP_RANGE_LOWER_BOUND>> <<<OPERATOR_BUNDLE_VERSION>>'
+    "olm.properties": '[{"type": "olm.maxOpenShiftVersion", "value": "4.8"}]'
```

Comment:
This is the format (long form, JSON stype?) provided by the OLM / Operators team.